app/cells/plugins/core/float_cell.rb

```diff
@@ -0,0 +1,57 @@
+module Plugins
+  module Core
+    class FloatCell < Plugins::Core::Cell
+      
+      def input
+        render
+      end
+     
+      private
+      
+      def max
+        field.validations[:max]  
+      end
+
+      def min
+        field.validations[:min] 
+      end
+
+      def step
+        decimals = field.validations[:decimals]
+        decimals ?  "0." + ("0" * ( decimals - 1)) + "1" :   "0.01"
```

Comment:
Howdy! We talked about this in person, but let's rebuild `step` to just take in the decimal step directly from the config - so the config would define `0.01` or `0.05` directly, rather than passing in the amount of sig digits to step by.